setup_telegram.py

```diff
@@ -0,0 +1,11 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "infomate.settings")
+import django
+django.setup()
+
+from telethon.sync import TelegramClient
+from django.conf import settings
+
+
+with TelegramClient(settings.TELEGRAM_SESSION_FILE, settings.TELEGRAM_APP_ID, settings.TELEGRAM_APP_HASH) as client:
```

Comment:
Could you explain please why do we need this file and what does it do?
I can't see any scripts calling it.

Reply:
This file need for initialize Telegram session. For initialize Telegram session you must run this script manually. This need only first time, before launch application. 
1. You enter your phone number
2. You receive code in your Telegram account and send that code in console for initialize Telegram session